instrumentation/netty/netty-4.1/src/test/resources/logback.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think with the `AgentTestRunner` fix we shouldn't need to copy logback.xml around